Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM centos:centos7
+MAINTAINER Mozilla Corporation Telemetry Pipeline team
+
+# Install the appropriate software
+RUN yum -y update; yum clean all
+RUN yum -y install epel-release; yum clean all
+RUN yum -y install cmake3 cmake jq make luasandbox hindsight luasandbox-lfs luasandbox-lpeg luasandbox-rjson luasandbox-cjson luasandbox-parquet; yum clean all
```

Comment:
Apparently some RPM packages cannot be found:

```
No package luasandbox available.
No package hindsight available.
No package luasandbox-lfs available.
No package luasandbox-lpeg available.
No package luasandbox-rjson available.
No package luasandbox-cjson available.
No package luasandbox-parquet available.
```

@trink do you know what I should do to install them?